### PR TITLE
docs: Desktop access: Move local Windows users content changes to a new version of the gett…

### PR DESCRIPTION
**Pull request by lsgunn-teleport**
…ing-started.mdx

Changes the title of the topic but not the file name and incorporates all of the changes from the old PR [#30707].

### CODE REVIEW

File: docs/pages/desktop-access/getting-started.mdx [hunk 1]
```diff
@@ -1,199 +1,237 @@
 ---
-title: Getting Started with Desktop Access
-description: Passwordless access to Windows systems for local users
+title: Configure access for local Windows users
+description: Configure passwordless access for local Windows users on Windows computers that aren't joined to an Active Directory domain.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:3]:
```suggestion
description: Configure passwordless access for local Windows users.
```

Technically it does work on domain-joined hosts too.

File: docs/pages/desktop-access/getting-started.mdx [hunk 2]
```diff
@@ -1,199 +1,237 @@
...
 videoBanner: 9DyKQbg4ORc
 ---
 
-This guide will help you configure Teleport to provide secure, passwordless access
-to Windows desktops. This configuration does not require an Active Directory domain.
+This guide demonstrates how to configure Teleport to provide secure, passwordless access
+to Microsoft Windows desktops that aren't joined to an Active Directory domain.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:8]: Is the description rendered on the page already? If it is, we probably don't need to say this twice.

File: docs/img/desktop-access/windows-local-users.png
Comment (zmb3) [inline docs/img/desktop-access/windows-local-users.png:1]: I don't love this diagram. I don't really know how to describe it, it just doesn't look clean to me and nothing here is really specific to local users (you could show the same diagram for AD users)

Can we get away without this and just put a more general architecture diagram on the overview page?

File: docs/cspell.json [hunk 3]
```diff
@@ -47,6 +47,7 @@
     "Dfumu",
     "Distroless",
     "Divio's",
+    "Doubleclick",
```
Comment (zmb3) [inline docs/cspell.json:50]: Is this actually a word? Isn't it `double-click`?

File: docs/pages/desktop-access/getting-started.mdx [hunk 4]
```diff
@@ -1,199 +1,237 @@
...
-  You can reuse an existing server running any other Teleport instance.
-- A server or virtual machine running a Windows operating system with
-  Remote Desktop enabled and the RDP port available to the Linux server.
+- Access to a running Teleport cluster.
+- A Linux server to run the Teleport Windows Desktop Service.
+  You can reuse an existing server running any other Teleport instance. 
+  However, you must generate a new invitation token to add the Windows Desktop Service
+  to an existing Teleport instance.
+- A physical or virtual machine running Microsoft Windows with Remote Desktop enabled 
+  and the RDP port—typically, port 3389—open for inbound connections from the Linux server.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:25]:
```suggestion
  and the RDP port (typically 3389) open for inbound connections from the Linux server.
```

File: docs/pages/desktop-access/getting-started.mdx [hunk 5]
```diff
@@ -1,199 +1,237 @@
...
+To prepare a Windows computer:
 
-Export the Teleport user certificate authority by running the following from
-`cmd.exe` on your Windows system:
+1. Open a Command Prompt (`cmd.exe`) window on the Windows computer you want to enroll.
 
-```code
-$ curl -o teleport.cer https://teleport-proxy.example.com/webapi/auth/export?type=windows
-```
+1. Export the Teleport user certificate authority by running the following command:
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:39]:
```suggestion
1. Export the Teleport user certificate authority by running the following command, replacing teleport-proxy.example.com with the address of your proxy server:
```

(or use a Var component)

File: docs/pages/desktop-access/getting-started.mdx [hunk 6]
```diff
@@ -1,199 +1,237 @@
...
 
-<Admonition type="tip" title="Headless Installation">
-The Teleport Windows Auth Setup can be run in a shell environment with elevated
-privileges. You can use this to automate installation, uninstallation, and
-certificate updates. The following command will install the Teleport Certificate,
-load the required DLL, disable NLA, and reboot the Windows machine:
+   The setup program:
+
+   - Installs the selected Teleport certificate.
+   - Loads the required dynamic link library (DLL) for Teleport to use.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:53]:
```suggestion
   - Installs the required dynamic link library (DLL) for Teleport to use.
```

File: docs/pages/desktop-access/getting-started.mdx [hunk 7]
```diff
@@ -1,199 +1,237 @@
...
+interactively and select the Teleport certificate that you exported when prompted.
 
-<Admonition type="tip" title="Headless Installation">
-The Teleport Windows Auth Setup can be run in a shell environment with elevated
-privileges. You can use this to automate installation, uninstallation, and
-certificate updates. The following command will install the Teleport Certificate,
-load the required DLL, disable NLA, and reboot the Windows machine:
+   The setup program:
+
+   - Installs the selected Teleport certificate.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:52]:
```suggestion
   - Configures Windows to trust the Teleport CA
```

File: docs/pages/desktop-access/getting-started.mdx [hunk 8]
```diff
@@ -1,199 +1,237 @@
...
+   - Loads the required dynamic link library (DLL) for Teleport to use.
+   - Disables Network Level Authentication (NLA) for remote desktop services. 
+
+1. Restart the computer.
+
+
+After you download the setup program so it's available on the Windows computer,
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:59]: I don't think we need to say 'after you downloaded the program' - we already told the user to download it and double click it.

We probably just want to say something like "If you prefer an automated install, you can invoke the installer from an administrative command prompt"

File: docs/pages/desktop-access/getting-started.mdx [hunk 9]
```diff
@@ -1,199 +1,237 @@
...
+
+1. Configure the `/etc/teleport.yaml` file on the Linux server with settings similar to the 
+following for the Windows Desktop Service: 
 
-</Details>
+   ```yml
+   version: v3
+   teleport:
+     nodename: <Var name="windows.teleport.example.com" />
+     proxy_server: <Var name="teleport.example.com:443" />
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:117]: Whatever we do, let's make the default value for proxy_server here match what was used in the curl command above.

File: docs/pages/desktop-access/getting-started.mdx [hunk 10]
```diff
@@ -1,199 +1,237 @@
...
+     auth_token: <Var name="/tmp/token" />
+   windows_desktop_service:
+     enabled: yes
+     non_ad_hosts:
+     - 192.0.2.156
+   auth_service:
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:123]:
```suggestion

   auth_service:
```

I find it helpful to separate the block that is just disabling other services.

File: docs/pages/desktop-access/getting-started.mdx [hunk 11]
```diff
@@ -1,199 +1,237 @@
...
+
+   If Teleport isn't installed on the Linux server, follow the appropriate [Installation
+instructions](../installation.mdx#installation-instructions) for your environment.
+
+1. Configure the `/etc/teleport.yaml` file on the Linux server with settings similar to the 
+following for the Windows Desktop Service: 
 
-</Details>
+   ```yml
+   version: v3
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:114]: YAML is pretty sensitive to whitespace. I see you indented this a bit make it "under" this step in the ordered list, but I just want to make sure that when this renders there won't be 3 leading spaces.

File: docs/pages/desktop-access/getting-started.mdx [hunk 12]
```diff
@@ -1,199 +1,237 @@
...
+   ssh_service:
+     enabled: no
+   ```
 
-(!docs/pages/includes/start-teleport.mdx service="the Teleport Desktop Service"!)
+   In this file:
 
-## Step 3/4. Configure Windows access
+   - Set the `proxy_server` to the address of your Teleport cluster.
+   - List the Windows IP addresses or DNS names under `non_ad_hosts`.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:134]:
```suggestion
   - List the Windows IP addresses or hostnames under `non_ad_hosts`.
```

DNS is the mechanism by which a hostname is resolved to an IP address, so I would prefer using hostname over "DNS name"

File: docs/pages/desktop-access/getting-started.mdx [hunk 13]
```diff
@@ -1,199 +1,237 @@
...
-In order to gain access to a remote desktop, a Teleport user needs to have the
+   Teleport can't automatically discover Windows desktops without Active Directory. 
+   Therefore, you must specify the Windows IP addresses or DNS names that you want to 
+   enroll using the Teleport configuration file or use the Teleport API  
+   to build your own integration. You can find an example of API integration on 
+   [GitHub](https://github.com/gravitational/teleport/tree/master/examples/desktop-registration).
+
+1. (Optional) Add labels for the Windows Desktop Service to the configuration file.
+
+   You can attach labels to Windows computers by matching to their hostnames.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:144]: We have a whole separate section on labeling in the RBAC page, does it make sense to link that here?

File: docs/pages/desktop-access/getting-started.mdx [hunk 14]
```diff
@@ -1,199 +1,237 @@
...
+   For example, the following adds the `cloud: ec2` label to computers that have 
+   EC2 private IP DNS names matching `.us-east-2.compute.internal`:
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:146]:
```suggestion
   hostnames ending with `.us-east-2.compute.internal`:
```

"EC2 private IP DNS names" is a mouthful.

File: docs/pages/desktop-access/getting-started.mdx [hunk 15]
```diff
@@ -1,199 +1,237 @@
...
+       windows_desktop_logins: ["Administrator", "alice"]
+   ```
 
-```code
-$ tctl create -f windows-desktop-admins.yaml
-```
+   In this file:
+
+   - Set `windows_desktop_labels` to specify any labels you want to use to allow or deny access to specific hosts.
+   - Set `windows_desktop_logins` to specify the Windows logins that Teleport users assigned to the role
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:192]: Maybe we should also expand on automatic user creation (the logins you specify in `windows_desktop_logins` don't have to exist ahead of time on the Windows machine, Teleport can create them on demand).

This is only true for non-AD hosts and only if RBAC allows it, and it's not well documented.

File: docs/pages/desktop-access/getting-started.mdx (hunk 2 above)
Comment (lsgunn-teleport) [reply docs/pages/desktop-access/getting-started.mdx:8]: The description frontmatter is used by web crawlers, not rendered as text on the page (that's probably configurable but not typical).

File: docs/cspell.json (hunk 3 above)
Comment (lsgunn-teleport) [reply docs/cspell.json:50]: I thought double-click looked a little old-fashioned. But it seems like "double click" and "double-click" are still preferred.

File: docs/pages/desktop-access/getting-started.mdx (hunk 11 above)
Comment (lsgunn-teleport) [reply docs/pages/desktop-access/getting-started.mdx:114]: It seems like the copy button strips out the extra space.

**Review by ptgott — APPROVED** [review]

**Review by stevenGravy — APPROVED** [review]

**Pull request merged**